R/bias.R

```diff
@@ -95,104 +93,104 @@ bias_sample <- function(true_values, predictions) {
 #'
 #' where \eqn{Q_t} is the set of quantiles that form the predictive
 #' distribution at time \eqn{t}. They represent our
-#' belief about what the true value $x_t$ will be. For consistency, we define
+#' belief about what the observed value $x_t$ will be. For consistency, we define
 #' \eqn{Q_t} such that it always includes the element
 #' \eqn{q_{t, 0} = - \infty} and \eqn{q_{t,1} = \infty}.
 #' \eqn{1()} is the indicator function that is \eqn{1} if the
 #' condition is satisfied and $0$ otherwise. In clearer terms, \eqn{B_t} is
 #' defined as the maximum percentile rank for which the corresponding quantile
-#' is still below the true value, if the true value is smaller than the
-#' median of the predictive distribution. If the true value is above the
+#' is still below the observed value, if the observed value is smaller than the
+#' median of the predictive distribution. If the observed value is above the
 #' median of the predictive distribution, then $B_t$ is the minimum percentile
 #' rank for which the corresponding quantile is still larger than the true
-#' value. If the true value is exactly the median, both terms cancel out and
+#' value. If the observed value is exactly the median, both terms cancel out and
 #' \eqn{B_t} is zero. For a large enough number of quantiles, the
 #' percentile rank will equal the proportion of predictive samples below the
-#' observed true value, and this metric coincides with the one for
+#' observed value, and this metric coincides with the one for
 #' continuous forecasts.
 #'
 #' Bias can assume values between
 #' -1 and 1 and is 0 ideally (i.e. unbiased).
-#' @param predictions vector of length corresponding to the number of quantiles
+#' @param predicted vector of length corresponding to the number of quantiles
 #' that holds predictions
```

Comment:
hold?

Reply:
I think currently this may be entirely incorrect - given that everything surrounding quantiles is weird. I created a super-issue for that: #351